.github/docker/testing/Dockerfile.testing-plugins-alma8

```diff
@@ -0,0 +1,69 @@
+ARG REGISTRY_URL
+
+FROM ${REGISTRY_URL}/almalinux:8
+
+RUN bash -e <<EOF
+
+dnf install -y 'dnf-command(config-manager)' epel-release zstd jq
+dnf config-manager --set-enabled powertools
+dnf clean all
+
+# Install Robotframework
+dnf install -y python3.11 python3.11-pip
+pip3.11 install robotframework robotframework-examples
+# Install snmpsim
+pip3.11 install snmpsim-lextudio
+
+# Install node
+curl -fsSL https://rpm.nodesource.com/setup_21.x | bash -
+yum install -y nodejs
+# Install mockoon
+npm install -g -D @mockoon/cli@7.0.0
```

Comment:
@sc979 is there a way to variabilize the version of the dependencies in a dockerfile via workflows and/or processes that we already have? (e.g. dependabot)

Reply:
CTOR-564 created to address this problem.

Reply:
Everything is always possible. But why should we variabilize this testing image's dependency version ?

Reply:
tested the last version of mockoon, I didn't see any bug.
I let the last mockoon version in the build of images so it will be updated automaticly on image rebuild.